gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/common/impl/ConfigStoreBackedTopology.java

```diff
@@ -0,0 +1,66 @@
+package gobblin.config.common.impl;
+
+import gobblin.config.store.api.ConfigKeyPath;
+import gobblin.config.store.api.ConfigStore;
+import gobblin.config.store.api.ConfigStoreWithImportedBy;
+import gobblin.config.store.api.ConfigStoreWithImportedByRecursively;
+import gobblin.config.store.api.ConfigStoreWithResolution;
+
+import java.util.Collection;
```

Comment:
Please fix the order of imports in your Eclipse config. java packages should be before gobblin ones.